packages/components/components/elvis-dropdown/src/react/dropdown-item/dropdownItem.tsx

```diff
@@ -156,6 +156,16 @@ export const DropdownItem: React.FC<DropdownItemProps> = ({
     };
   }, []);
 
+  const lang = useLanguage();
+  const translatedAriaLabel = () => {
+    return {
+      submenu: lang === 'no' ? 'undermeny' : 'submenu',
+      note: lang === 'no' ? 'Merknad:' : 'Note:',
+    };
+  };
+
+  const labels = translatedAriaLabel();
```

Comment:
Any reason it isn't done like this? Why the extra function `translatedAriaLabel`?
```suggestion
  const labels = {
      submenu: lang === 'no' ? 'undermeny' : 'submenu',
      note: lang === 'no' ? 'Merknad:' : 'Note:',
    };```

Reply:
I made the changes you suggested.